### DISCUSSION TITLE
Best practice for Pimcore Core Development?

### DISCUSSION THREAD

**Discussion by dpfahlbusch** [pimcore/pimcore#12109] [Q&A / Support] [answered]
Hey all,
are there any best practices for Pimcore Core Development?

Like checking out the skeleton and then just start working?
Could need some tipps and tricks, because Currently I have the problem that the javascript files are cached and my changes have no effect.

**Comment by dvesh3** [comment, marked as answer]
please try setting env var `PIMCORE_DEV_MODE=true`

**Reply by dpfahlbusch** [reply]
`PIMCORE_DEV_MODE=true` was set.
The problem was Chrome. The javascript files were aggressively cached.
Problem solved via:
Opening the dev tools and used the "Clear cache and reload page" feature of chrome with open dev-tools
